doc/source/PreConfiguredSites.rst

```diff
@@ -539,7 +539,7 @@ For ``spack-stack-1.5.0`` with Intel, load the following modules after loading m
 
 .. code-block:: console
 
-   module use /scratch1/NCEPDEV/nems/role.epic/spack-stack/spack-stack-1.5.0/envs/unified-env/install/modulefiles/Core
+   module use /scratch1/NCEPDEV/nems/role.epic/spack-stack/spack-stack-1.5.0/envs/unified-env-noavx512/install/modulefiles/Core
```

Comment:
do these `noavx512` installations correspond to the updated haswell target change for hera? should these installations now be used on all nodes?

Reply:
Yes and yes, I made the change in my ufs-weather-model PR, too.